### PR TITLE
Add e6data query runner

### PR DESCRIPTION
**Pull request by anushae6**
## What type of PR is this? 

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [X] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## How is this tested?

- [X] Unit tests (pytest, jest)
- [X] E2E Tests (Cypress)
- [ ] Manually
- [ ] N/A

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -86,6 +86,7 @@ werkzeug = "2.3.6"
 wtforms = "2.2.1"
 xlsxwriter = "1.2.2"
 tzlocal = "4.3.1"
+e6data-python-connector = "1.1.9"
```
Comment (konnectr) [inline pyproject.toml:89]: You should add this to [tool.poetry.group.all_ds.dependencies]

File: redash/query_runner/e6data.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+                message = None
+            error = default_message if message is None else message
+        except (KeyboardInterrupt, InterruptException, JobTimeoutException) as intExp:
+            error = intExp
+            json_data = None
+            cursor.cancel()
+        except Exception:
+            error = "Invalid Configuration"
+            json_data = None
+        return json_data, error
```
Comment (konnectr) [inline redash/query_runner/e6data.py:125]: Strange work with error

File: pyproject.toml (hunk 1 above)
Comment (masayuki038) [reply pyproject.toml:89]: @konnectr Nice catch! I missed it 😅

Comment (anushae6) [reply pyproject.toml:89]: > You should add this to [tool.poetry.group.all_ds.dependencies] and remove from here

Alright, will do that. Thanks

**Comment by guidopetri** [comment]
Duplicated in #6604

**Pull request closed without merging**